src/ast/nodes/MemberExpression.ts

```diff
@@ -78,7 +78,8 @@ export default class MemberExpression extends NodeBase {
 				this.replacement = resolvedVariable;
 			} else {
 				if (isExternalVariable(resolvedVariable)) {
-					resolvedVariable.module.suggestName(path[0].key);
+					if (resolvedVariable.module)
+						resolvedVariable.module.suggestName(path[0].key);
```

Comment:
We could actually merge the innermost if-statements? 😉

Reply:
Sure, included.